tests/qsfs_check_metrics/qsfs_test.go

```diff
@@ -42,25 +39,23 @@ func TestMultiNodeDeployment(t *testing.T) {
 	// Check that the outputs not empty
 	metrics := terraform.Output(t, terraformOptions, "metrics")
 	assert.NotEmpty(t, metrics)
+	metrics, err := tests.IPFromCidr(metrics)
```

Comment:
metrics is something like `http://[300:9c4a:29ac:6e0b:19e2:80f1:de46:215f]:9100/metrics` so parsing it like that returns an error